pkg/model/stack_expander_test.go

```diff
@@ -22,19 +22,22 @@ services:
     myservice:
         build:
             context: vote
-        args:
-            - $CUSTOM_ENV
-            - CUSTOM2_ENV=$CUSTOM_ENV
-            - EMPTY
+            args:
+                - ENV=hello
+                - CUSTOM2_ENV=$CUSTOM_ENV
+                - EMPTY=
+                - ENV2=
```

Comment:
should this one translate to `ENV2=${ENV2}`? MAybe that only works for `environment`

Reply:
the stack expander only is responsible for expand the envs. The logic that makes this happen is at the stack unmarshall, where we check that if an env only has key, and there is a value for this key as env, we assing this value